### DISCUSSION TITLE
Should we use the @tailwindcss/ui plugin even if we don't use TailwindUI?

### DISCUSSION THREAD

**Discussion by jalvarado91** [tailwindlabs/tailwindcss#2048] [Help] [answered]
Like the title says. Should people use the @tailwindcss/ui plugin even if we aren't using the TailwindUI product? As I understand, the @tailwindcss/ui plugin mostly extends the default tailwind config with stuff that's going to make it to tailwind itself and a lot of the utilities and extensions are great for, well, building UIs. 

Is there a licensing breach if one does do this? If not, I don't see a reason not to add it. And I've seen projects use it for the same reason of taking advantage of the extensions.

**Comment by mgibas** [comment]
`@tailwindcss/ui` has MIT license: https://www.npmjs.com/package/@tailwindcss/ui

**Comment by adamwathan** [comment, marked as answer]
Can definitely use it anywhere you like, I think in the near-ish future we will rename it to be more of a "2.0 preview" plugin or find some way to integrate the changes directly into Tailwind in an opt-in way.
